? unowinreg-windows-h-include.diff
? win/1
? win/autom4te.cache
? win/autoscan.log
? win/build.sh
? win/build.sh.in
? win/config.log
? win/config.status
? win/configure
? win/configure.in
? win/unowinreg.dll
Index: win/unowinreg.c
===================================================================
RCS file: /cvs/api/odk/source/unowinreg/win/Attic/unowinreg.c,v
retrieving revision 1.3
diff -u -u -r1.3 unowinreg.c
--- odk/source/unowinreg/win/unowinreg.c	7 Sep 2005 20:21:07 -0000	1.3
+++ odk/source/unowinreg/win/unowinreg.c	16 Jul 2006 08:31:27 -0000
@@ -33,7 +33,7 @@
  *
  ************************************************************************/
 
-#include <Windows.h>
+#include <windows.h>
 #include <jni.h>
 
 BOOL __stdcall _DllMainCRTStartup(
